### PR TITLE
RubyCats blogpost about Euruko and Codemotion Warsaw with images

### PR DESCRIPTION
**Pull request by agnikka**

### CODE REVIEW

File: blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md
Comment (alicetragedy) [inline blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md]: please change the filename to have the correct date (same as permalink and `created_at`)

File: blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md
Comment (alicetragedy) [inline blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md]: as explained [in our contributing guide](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images) all images need to have a credit (i.e. who took the pictures / on which site the picture was found). Could you add this to all images in the post after the caption? You can see examples for this in past posts :)

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @agnik5, thank you for the post, it's great! I've made comments of a couple of things that need changing before we can merge your PR. Could you make those changes today still, so we can publish it? 🎉

File: blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md
Comment (agnikka) [reply blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md]: Hi @alicetragedy I'm very sorry for late reply, I spent almost the whole flu-day in bed and getting back to life and work. Thank you for your feedback :)
Sorry for a typo, of course I'll correct it.
All images were taken by us instead of the one from our LT, which was taken by Team Joda. I'll add this info.
As it's quite late, should I change both dates /filename and permalink/ to 30 Sept?

Comment (alicetragedy) [reply blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md]: hey @agnik5 yes please add this info to all images and make the corrections. Please keep the date for today, we already have posts scheduled for tomorrow and Saturday. Maybe we could also post in on Sunday if you prefer?

Comment (agnikka) [reply blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md]: It depends on your preference :) Those changes won't take me a long time :)

**Comment by alicetragedy** [comment]
@agnik5 then let's try to ship it today! if that doesn't work, I will change the date myself, no worries 🎉

**Review by alicetragedy — APPROVED** [review]
ready to merge!!

**Pull request merged**
